src/enums/HttpErrorStatusCode.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * StatusCode Enumerator
+ * Define http codes returned by controllers
+ */
+
+export enum HttpErrorStatusCode {
```

Comment:
we already have [this enum](https://github.com/teamdigitale/italia-ts-commons/blob/master/src/responses.ts#L8)